classindex/pom.xml

```diff
@@ -127,6 +127,18 @@
 					<testTarget>8</testTarget>
 				</configuration>
 			</plugin>
+            <plugin>
```

Comment:
This file uses tab for indentation, please change spaces to tabs for consistency

Reply:
You're right. Sorry. It's done.